packages/native_pdf_renderer/lib/src/page_image.dart

```diff
@@ -70,14 +76,21 @@ class PdfPageImage {
       'crop_y': crop?.top?.toInt(),
       'crop_height': crop?.height?.toInt(),
       'crop_width': crop?.width?.toInt(),
+      'quality': quality,
     });
 
     if (!(obj is Map<dynamic, dynamic>)) {
       return null;
     }
 
     final retWidth = obj['width'] as int, retHeight = obj['height'] as int;
-    final pixels = Uint8List.fromList(obj['data']);
+    final pixels = Platform.isAndroid

```

Comment:
Why did you decide to save the file? 
Does this give any performance boost?

Reply:
about performance ok, looked https://github.com/rbcprolabs/packages.flutter/issues/150#issue-793336180